src/main/scala/io/qbeast/catalog/QbeastCatalogUtils.scala

```diff
@@ -110,21 +106,15 @@ object QbeastCatalogUtils extends Logging {
     }
   }
 
-  private def verifySchema(
-      spark: SparkSession,
-      fs: FileSystem,
-      path: Path,
-      table: CatalogTable): CatalogTable = {
+  private def verifySchema(indexedTable: IndexedTable, table: CatalogTable): CatalogTable = {
```

Comment:
We should add some comments about what is the scope of this method (I know it should've been written before, sorry for that :((( )

Reply:
Added